### PR TITLE
Login bugfix

### PR DESCRIPTION
**Pull request by diamond-darrell**
### PR information
**What kind of change does this PR introduce?** (check one with "x")
- [x] Bugfix
- [] Feature
- [] Code style update (formatting, local variables)
- [] Refactoring (no functional changes, no api changes)
- [] Build related changes
- [] CI related changes
- [] Other... Please describe:

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**
Fixes of:
- KMCNG-835
- KMCNG-834
- KMCNG-838
- KMCNG-831
- KMCNG-832

**Does this PR introduce a breaking change?** (check one with "x")
- [] Yes
- [] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

---

### Other information

**Where should the reviewer start?**

**How should this be manually tested?**

**Any background context you want to provide?**

---
This change is [Reviewable](https://reviewable.io/reviews/kaltura/kmc-ng/247)

### CODE REVIEW

File: src/app/components/login/login.component.ts [hunk 1]
```diff
@@ -28,11 +27,30 @@ export class LoginComponent implements OnInit, OnDestroy {
   public _currentScreen = LoginScreens.Login;
   public _passwordReset = false;
 
+  // Caution: this is extremely dirty hack, don't do something similar to that, damn you IE
+  @HostListener('window:resize')
+  onResize() {
+    const areaBlocker = <any>document.querySelector('k-area-blocker');
```
Comment (amirch1) [inline src/app/components/login/login.component.ts]: Lets wrap with isIE11 from browser service

**Comment by diamond-darrell** [comment]
Review status: 0 of 11 files reviewed at latest revision, 1 unresolved discussion.

---

*[src/app/components/login/login.component.ts, line 33 at r1](https://reviewable.io:443/reviews/kaltura/kmc-ng/247#-KxTF3ciES_Eu6nLF31E-r1-33:-KyG6BhpdWeGS5XqQ_XC:b32w0x) ([raw file](https://github.com/kaltura/kmc-ng/blob/a9895ae0f0ada4812ec6cfada8f03d3190debe0e/src/app/components/login/login.component.ts#L33)):*
<details><summary><i>Previously, amirch1 (Amir Chervinsky) wrote…</i></summary><blockquote>

Lets wrap with isIE11 from browser service
</blockquote></details>

OK

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/kaltura/kmc-ng/247)*

**Comment by amirch1** [comment]
Reviewed 4 of 8 files at r1, 7 of 7 files at r2.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/kaltura/kmc-ng/247)*

**Comment by amirch1** [comment]
Review status: all files reviewed at latest revision, 1 unresolved discussion, all commit checks successful.

---

*[src/app/components/login/login.component.ts, line 56 at r2](https://reviewable.io:443/reviews/kaltura/kmc-ng/247#-KyK6cLk8xoCyUGzczA3:-KyK6cLl-kSnMxvmDIIz:b-f1xnpq) ([raw file](https://github.com/kaltura/kmc-ng/blob/4aa30880489312d87276f2b36e97bf0c75899113/src/app/components/login/login.component.ts#L56)):*
> ```XML
>   }
> 
>   ngAfterViewInit() {
> ```

seems you also need to call resize when switching from login to "forgot password" else the screen size is wrong (you see a white section at the bottom). So maybe move this logic to a function and call it in AfterViewInit and whenever switching between login screens. 

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/kaltura/kmc-ng/247)*

**Comment by diamond-darrell** [comment]
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

*[src/app/components/login/login.component.ts, line 56 at r2](https://reviewable.io:443/reviews/kaltura/kmc-ng/247#-KyK6cLk8xoCyUGzczA3:-KyKEoNzX1QUeh5DE-sR:b32w0x) ([raw file](https://github.com/kaltura/kmc-ng/blob/4aa30880489312d87276f2b36e97bf0c75899113/src/app/components/login/login.component.ts#L56)):*
<details><summary><i>Previously, amirch1 (Amir Chervinsky) wrote…</i></summary><blockquote>

seems you also need to call resize when switching from login to "forgot password" else the screen size is wrong (you see a white section at the bottom). So maybe move this logic to a function and call it in AfterViewInit and whenever switching between login screens. 
</blockquote></details>

OK

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/kaltura/kmc-ng/247)*

**Comment by diamond-darrell** [comment]
Review status: 10 of 11 files reviewed at latest revision, 2 unresolved discussions.

---

*[src/app/components/login/login.component.ts, line 57 at r2](https://reviewable.io:443/reviews/kaltura/kmc-ng/247#-KyKFkOfjUvMAhiHEkin:-KyKFkOfjUvMAhiHEkio:b-7rg0tl) ([raw file](https://github.com/kaltura/kmc-ng/blob/4aa30880489312d87276f2b36e97bf0c75899113/src/app/components/login/login.component.ts#L57)):*
> ```XML
> 
>   ngAfterViewInit() {
>     if (this._browserService.isIE11()) {
> ```

I've removed this `if` because `onResize` already has it 

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/kaltura/kmc-ng/247)*

**Comment by amirch1** [comment]
:lgtm:

---

Reviewed 1 of 1 files at r3.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/kaltura/kmc-ng/247#-:-KyKXqD8mKJE0uc54BCh:bnfp4nl)*

**Comment by esakal** [comment]
Reviewed 4 of 8 files at r1, 6 of 7 files at r2, 1 of 1 files at r3.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/kaltura/kmc-ng/247)*

**Comment by esakal** [comment]
:lgtm:

---

Review status: :shipit: all files reviewed at latest revision, all discussions resolved, all commit checks successful.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/kaltura/kmc-ng/247#-:-KyPmf2EAPqXQTQLPM3S:bnfp4nl)*

**Pull request merged**